xxh3.h

```diff
@@ -633,7 +633,7 @@ XXH_FORCE_INLINE void XXH3_initKeySeed(U32* key, U64 seed64)
 XXH_NO_INLINE XXH64_hash_t    /* It's important for performance that XXH3_hashLong is not inlined. Not sure why (uop cache maybe ?), but difference is large and easily measurable */
 XXH3_hashLong_64b(const void* data, size_t len, XXH64_hash_t seed)
 {
-    ALIGN(64) U64 acc[ACC_NB] = { seed, PRIME64_1, PRIME64_2, PRIME64_3, PRIME64_4, PRIME64_5, (U64)0 - seed, 0 };
+    ALIGN(64) U64 acc[ACC_NB] = { seed, PRIME64_1, PRIME64_2, PRIME64_3, PRIME64_4, PRIME64_5, (U64)0 - seed, 0 }; /* fixme: warning C4204: nonstandard extension used: non-constant aggregate initializer */
```

Comment:
👍

Reply:
I do want to mention that MSVC will not let you use static const variables as constant expressions in `/Ta` or whatever the flag is for ANSI conformance.

However, agreed that seed should be inserted manually.

Reply:
I need to add a test to ensure such issues are caught during CI tests.